dashboards/sonarqube/sonarqube.json

```diff
@@ -35,7 +35,7 @@
 						"id": "viz.markdown"
 					},
 					"rawConfiguration": {
-						"text": "\n\n**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/marketplace?account=3627899&state=70385b17-af48-cb31-24f8-fa1e82a0ea5f).\n\nUnable to find data in your dashboard? -[Troubleshoot here](\nhttps://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/configure-prometheus-openmetrics-integrations/#general-config)\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=SonarQube) here and let us know how we can improve it for you.\n"
+						"text": "\n\n**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/marketplace?account=3510613&state=7f6689ac-8fd7-b1c2-3c79-d64f0cecee2d).\n\nUnable to find data in your dashboard? -[Troubleshoot here](\nhttps://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/configure-prometheus-openmetrics-integrations/#general-config)\n\nFollow New Relic [SonaQube documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration/) to instrument SonarQube.\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=SonarQube) here and let us know how we can improve it for you.\n"
```

Comment:
```suggestion
						"text": "\n\n**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details?state=337b820d-ada6-527d-77eb-90faed3fd418).\n\nUnable to find data in your dashboard? -[Troubleshoot here](\nhttps://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/configure-prometheus-openmetrics-integrations/#general-config)\n\nFollow New Relic [SonaQube documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration/) to instrument SonarQube.\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=SonarQube) here and let us know how we can improve it for you.\n"
```

Reply:
Hi @pkosanam ,
As per your suggestion updated the add data URL. Kindly review it.

Thanks.